sb_vision/tokens.py

```diff
@@ -51,57 +50,57 @@ def from_apriltag_detection(
     ) -> 'Token':
         """Construct a Token from an April Tag detection."""
 
-        pixel_coords = [Coordinate(*l) for l in apriltag_detection.p]
+        pixel_corners = [PixelCoordinate(*l) for l in apriltag_detection.p]
 
         # centre of marker: average the corners
-        pixel_centre = Coordinate(*np.average(pixel_coords, axis=0))
+        pixel_centre = PixelCoordinate(*np.average(pixel_corners, axis=0))
 
         marker_id = apriltag_detection.id
 
         # *************************************************************************
         # NOTE: IF YOU CHANGE THIS, THEN CHANGE ROBOT-API camera.py
         # *************************************************************************
+
+        # The pixel_corners value we expose has the first one in the bottom left.
+        # Apriltags returns the first one in the top left. thus we need to rotate
+        # the values by one to get the correct ordering.
+        offset_pixel_corners = [pixel_corners[3]] + pixel_corners[:3]
```

Comment:
Can we clarify whether this the top left of the _marker_ (i.e: relative to what the marker defines as the "top"), or the top left of the _image_ of the marker (i.e: relative to the image itself). (Or something else)

Reply:
I think I've clarified it in a simple way in 40688997f88f59c26beb3a7689e3253f2f399149